diff -ur debrix/hw/xorg/common/Makefile.am debrix-new/hw/xorg/common/Makefile.am
--- debrix/hw/xorg/common/Makefile.am	2004-06-10 19:14:51.000000000 +0000
+++ debrix-new/hw/xorg/common/Makefile.am	2004-06-29 11:13:43.000000000 +0000
@@ -58,7 +58,7 @@
 INCLUDES = $(XORG_INCS) -I$(srcdir)/../loader -I$(srcdir)/../ddc \
            -I$(srcdir)/../rac -I$(srcdir)/../vbe -I$(srcdir)/../int10 \
            -I$(srcdir)/../i2c -I$(srcdir)/../parser -I$(srcdir)/../vgahw \
-           -I$(srcdir)/../scanpci -I$(top_srcdir)/Xi
+           -I$(srcdir)/../scanpci -I$(top_srcdir)/Xi -I$(srcdir)/..
 
 sdk_HEADERS = compiler.h fourcc.h xf86.h xf86Module.h xf86Opt.h \
                xf86PciInfo.h xf86Priv.h xf86Privstr.h xf86Resources.h \
diff -ur debrix/hw/xorg/common/xf86pciBus.c debrix-new/hw/xorg/common/xf86pciBus.c
--- debrix/hw/xorg/common/xf86pciBus.c	2004-06-10 19:40:04.000000000 +0000
+++ debrix-new/hw/xorg/common/xf86pciBus.c	2004-06-29 11:41:51.000000000 +0000
@@ -1709,25 +1709,12 @@
     /*
      * Initialise the pcidata entry points.
      */
-#ifdef XFree86LOADER
-    xf86SetupPciIds = (ScanPciSetupProcPtr)LoaderSymbol("ScanPciSetupPciIds");
-    xf86ClosePciIds = (ScanPciCloseProcPtr)LoaderSymbol("ScanPciClosePciIds");
-    xf86FindPciNamesByDevice =
-	(ScanPciFindByDeviceProcPtr)LoaderSymbol("ScanPciFindPciNamesByDevice");
-    xf86FindPciNamesBySubsys =
-	(ScanPciFindBySubsysProcPtr)LoaderSymbol("ScanPciFindPciNamesBySubsys");
-    xf86FindPciClassBySubsys =
-	(ScanPciFindClassBySubsysProcPtr)LoaderSymbol("ScanPciFindPciClassBySubsys");
-    xf86FindPciClassByDevice =
-	(ScanPciFindClassByDeviceProcPtr)LoaderSymbol("ScanPciFindPciClassByDevice");
-#else
     xf86SetupPciIds = ScanPciSetupPciIds;
     xf86ClosePciIds = ScanPciClosePciIds;
     xf86FindPciNamesByDevice = ScanPciFindPciNamesByDevice;
     xf86FindPciNamesBySubsys = ScanPciFindPciNamesBySubsys;
     xf86FindPciClassBySubsys = ScanPciFindPciClassBySubsys;
     xf86FindPciClassByDevice = ScanPciFindPciClassByDevice;
-#endif
 
     if (!xf86SetupPciIds())
 	FatalError("xf86SetupPciIds() failed\n");
diff -ur debrix/hw/xorg/ddc/Makefile.am debrix-new/hw/xorg/ddc/Makefile.am
--- debrix/hw/xorg/ddc/Makefile.am	2004-06-27 21:20:53.000000000 +0000
+++ debrix-new/hw/xorg/ddc/Makefile.am	2004-06-29 11:13:21.000000000 +0000
@@ -4,4 +4,4 @@
 libddc_a_SOURCES = xf86DDC.c edid.c interpret_edid.c print_edid.c \
                    interpret_vdif.c print_vdif.c ddcProperty.c
 
-INCLUDES = $(XORG_INCS) -I$(srcdir)/../i2c
+INCLUDES = $(XORG_INCS) -I$(srcdir)/..
diff -ur debrix/hw/xorg/ddc/xf86DDC.h debrix-new/hw/xorg/ddc/xf86DDC.h
--- debrix/hw/xorg/ddc/xf86DDC.h	2004-06-10 19:40:12.000000000 +0000
+++ debrix-new/hw/xorg/ddc/xf86DDC.h	2004-06-29 11:13:21.000000000 +0000
@@ -18,7 +18,7 @@
 # define XF86_DDC_H
 
 #include "edid.h"
-#include "xf86i2c.h"
+#include "i2c/xf86i2c.h"
 #include "xf86str.h"
 
 /* speed up / slow down */
diff -ur debrix/hw/xorg/int10/Makefile.am debrix-new/hw/xorg/int10/Makefile.am
--- debrix/hw/xorg/int10/Makefile.am	2004-06-27 21:20:53.000000000 +0000
+++ debrix-new/hw/xorg/int10/Makefile.am	2004-06-29 11:41:12.000000000 +0000
@@ -3,7 +3,8 @@
 #ibxorgint10_a_SOURCES = stub.c
 
 libint10_a_SOURCES = pci.c xf86int10module.c helper_exec.c helper_mem.c \
-                     xf86int10.c generic.c xf86int10module.c
+                     xf86int10.c generic.c xf86int10module.c sys.c \
+                     xf86x86emu.c decode.c prim_ops.c ops.c ops2.c fpu.c
 
 # FIXME: i386/amd64-only (and Linux-only!!)
 AM_CFLAGS = -D_PC -D_VM86_LINUX
diff -ur debrix/hw/xorg/loader/Makefile.am debrix-new/hw/xorg/loader/Makefile.am
--- debrix/hw/xorg/loader/Makefile.am	2004-06-28 10:25:18.000000000 +0000
+++ debrix-new/hw/xorg/loader/Makefile.am	2004-06-29 11:13:45.000000000 +0000
@@ -1,8 +1,6 @@
 noinst_LIBRARIES = libloader.a
 AM_CFLAGS = $(XORG_CFLAGS) -DDLOPEN_SUPPORT -DHAS_SHM -DXFree86LOADER
-INCLUDES = $(XORG_INCS) -I$(srcdir)/../parser -I$(srcdir)/../vbe \
-           -I$(srcdir)/../ddc -I$(srcdir)/../int10 -I$(srcdir)/../i2c \
-           -I$(srcdir)/../fbdevhw -I$(srcdir)/../xaa
+INCLUDES = $(XORG_INCS) -I$(srcdir)/../parser -I$(srcdir)/..
 
 libloader_a_SOURCES = \
         dlloader.c \
diff -ur debrix/hw/xorg/loader/xf86sym.c debrix-new/hw/xorg/loader/xf86sym.c
--- debrix/hw/xorg/loader/xf86sym.c	2004-06-28 10:25:18.000000000 +0000
+++ debrix-new/hw/xorg/loader/xf86sym.c	2004-06-29 11:13:45.000000000 +0000
@@ -77,17 +77,22 @@
 #include "xf86xvmc.h"
 #include "xf86cmap.h"
 #include "xf86fbman.h"
-#include "fbdevhw.h"
+#include "fbdevhw/fbdevhw.h"
+#include "vgahw/vgaHW.h"
+#include "int10/xf86int10.h"
+#include "i2c/xf86i2c.h"
+#include "ddc/xf86DDC.h"
+#include "ramdac/xf86Cursor.h"
 #include "dgaproc.h"
 #include "dpmsproc.h"
 #include "vidmodeproc.h"
 #include "xf86miscproc.h"
 #include "loader.h"
-#include "xaarop.h"
+#include "xaa/xaarop.h"
 #define DONT_DEFINE_WRAPPERS
 #include "xf86_ansic.h"
 #include "xisb.h"
-#include "vbe.h"
+#include "vbe/vbe.h"
 #ifndef __OpenBSD__
 #include "xf86sbusBus.h"
 #endif
@@ -1158,6 +1163,27 @@
     /* fbdevhw.c */
     SYMVAR(fbdevHWValidMode)
 
+    /* vgaHW.c */
+    SYMVAR(vgaHWInit)
+
+    /* ini10 */ 
+    SYMVAR(xf86int10Addr)
+
+    /* xf86i2c.c */
+    SYMVAR(xf86CreateI2CBusRec)
+
+    /* vbe.c */
+    SYMVAR(VBEInit)
+
+    /* xf86DDC.c */
+    SYMVAR(xf86DoEDID_DDC2)
+
+    /* print_edid.c */
+    SYMVAR(xf86PrintEDID)
+
+    /* xf86Cursor.c */
+    SYMVAR(xf86InitCursor)
+
     /* xaa */
     SYMVAR(XAACopyROP)
 
diff -ur debrix/hw/xorg/vbe/Makefile.am debrix-new/hw/xorg/vbe/Makefile.am
--- debrix/hw/xorg/vbe/Makefile.am	2004-06-27 21:20:53.000000000 +0000
+++ debrix-new/hw/xorg/vbe/Makefile.am	2004-06-29 11:13:22.000000000 +0000
@@ -1,6 +1,5 @@
 noinst_LIBRARIES = libvbe.a
 libvbe_a_SOURCES = vbe.c vbeModes.c vbe_module.c
-INCLUDES = $(XORG_INCS) -I$(srcdir)/../ddc -I$(srcdir)/../i2c \
-          -I$(srcdir)/../int10
+INCLUDES = $(XORG_INCS) -I$(srcdir)/..
 
 sdk_HEADERS = vbe.h vbeModes.h
diff -ur debrix/hw/xorg/vbe/vbe.h debrix-new/hw/xorg/vbe/vbe.h
--- debrix/hw/xorg/vbe/vbe.h	2004-06-10 19:41:27.000000000 +0000
+++ debrix-new/hw/xorg/vbe/vbe.h	2004-06-29 11:13:22.000000000 +0000
@@ -15,8 +15,8 @@
 #endif
 #ifndef _VBE_H
 #define _VBE_H
-#include "xf86int10.h"
-#include "xf86DDC.h"
+#include "int10/xf86int10.h"
+#include "ddc/xf86DDC.h"
 
 typedef enum {
     DDC_UNCHECKED,
diff -ur debrix/hw/xorg/vgahw/Makefile.am debrix-new/hw/xorg/vgahw/Makefile.am
--- debrix/hw/xorg/vgahw/Makefile.am	2004-06-27 21:20:54.000000000 +0000
+++ debrix-new/hw/xorg/vgahw/Makefile.am	2004-06-29 11:13:35.000000000 +0000
@@ -1,5 +1,5 @@
 noinst_LIBRARIES = libvgahw.a
 libvgahw_a_SOURCES = vgaHW.c vgaHWmodule.c
-INCLUDES = $(XORG_INCS) -I$(srcdir)/../ddc -I$(srcdir)/../i2c
+INCLUDES = $(XORG_INCS) -I$(srcdir)/..
 
 sdk_HEADERS = vgaHW.h
diff -ur debrix/hw/xorg/vgahw/vgaHW.h debrix-new/hw/xorg/vgahw/vgaHW.h
--- debrix/hw/xorg/vgahw/vgaHW.h	2004-06-10 19:41:30.000000000 +0000
+++ debrix-new/hw/xorg/vgahw/vgaHW.h	2004-06-29 11:13:35.000000000 +0000
@@ -26,7 +26,7 @@
 #include "xf86str.h"
 #include "xf86Pci.h"
 
-#include "xf86DDC.h"
+#include "ddc/xf86DDC.h"
 
 #include "globals.h"
 #define DPMS_SERVER
