Index: composite/compalloc.c
===================================================================
RCS file: /cvs/xserver/debrix/composite/compalloc.c,v
retrieving revision 1.7
diff -u -r1.7 compalloc.c
--- composite/compalloc.c	20 Nov 2003 03:31:29 -0000	1.7
+++ composite/compalloc.c	28 Jun 2004 00:03:04 -0000
@@ -57,6 +57,7 @@
     CompWindowPtr	cw = GetCompWindow (pWin);
     CompClientWindowPtr	ccw;
     Bool		wasMapped = pWin->mapped;
+    ScreenPtr		pScreen = pWin->drawable.pScreen;
 
     /*
      * Only one Manual update is allowed
@@ -91,6 +92,7 @@
 				   compDestroyDamage,
 				   DamageReportNonEmpty,
 				   FALSE,
+				   pScreen,
 				   pWin);
 	if (!cw->damage)
 	{
Index: hw/xorg/common/xf86KbdLnx.c
===================================================================
RCS file: /cvs/xserver/debrix/hw/xorg/common/xf86KbdLnx.c,v
retrieving revision 1.2
diff -u -r1.2 xf86KbdLnx.c
--- hw/xorg/common/xf86KbdLnx.c	10 Jun 2004 19:40:04 -0000	1.2
+++ hw/xorg/common/xf86KbdLnx.c	28 Jun 2004 00:03:05 -0000
@@ -73,7 +73,6 @@
 #include "atKeynames.h"
 
 #include "xf86Keymap.h"
-#include <X11/DECkeysym.h>
 
 /*
  * LegalModifier --
Index: hw/xorg/os-support/linux/lnx_KbdMap.c
===================================================================
RCS file: /cvs/xserver/debrix/hw/xorg/os-support/linux/lnx_KbdMap.c,v
retrieving revision 1.3
diff -u -r1.3 lnx_KbdMap.c
--- hw/xorg/os-support/linux/lnx_KbdMap.c	10 Jun 2004 19:40:55 -0000	1.3
+++ hw/xorg/os-support/linux/lnx_KbdMap.c	28 Jun 2004 00:03:05 -0000
@@ -24,7 +24,6 @@
 #include "atKeynames.h"
 
 #include "xf86Keymap.h"
-#include <X11/DECkeysym.h>
 
 #include "lnx_kbd.h"
 
Index: hw/xorg/os-support/shared/drm/kernel/drm.h
===================================================================
RCS file: /cvs/xserver/debrix/hw/xorg/os-support/shared/drm/kernel/drm.h,v
retrieving revision 1.2
diff -u -r1.2 drm.h
--- hw/xorg/os-support/shared/drm/kernel/drm.h	10 Jun 2004 19:41:11 -0000	1.2
+++ hw/xorg/os-support/shared/drm/kernel/drm.h	28 Jun 2004 00:03:05 -0000
@@ -41,7 +41,6 @@
 #define _DRM_H_
 
 #if defined(__linux__)
-#include <linux/config.h>
 #include <asm/ioctl.h>		/* For _IO* macros */
 #define DRM_IOCTL_NR(n)		_IOC_NR(n)
 #define DRM_IOC_VOID		_IOC_NONE
Index: os/xdmauth.c
===================================================================
RCS file: /cvs/xserver/debrix/os/xdmauth.c,v
retrieving revision 1.3
diff -u -r1.3 xdmauth.c
--- os/xdmauth.c	10 Jun 2004 19:41:59 -0000	1.3
+++ os/xdmauth.c	28 Jun 2004 00:03:06 -0000
@@ -52,7 +52,7 @@
 #ifdef XDMCP
 #include <X11/Xmd.h>
 #undef REQUEST
-#include "Xdmcp.h"
+#include <X11/Xdmcp.h>
 
 /* XDM-AUTHENTICATION-1 */
 
