--- debrix-old/hw/xorg/loader/xf86sym.c	2004-07-10 02:33:56.411606808 +0200
+++ debrix/hw/xorg/loader/xf86sym.c	2004-07-10 02:33:47.444969944 +0200
@@ -1209,6 +1209,7 @@
     /* fb */
     SYMVAR(fbWinPrivateIndex)
     SYMFUNC(fbScreenInit)
+    SYMFUNC(fbPictureInit)
 
     /* ramdac */
     SYMFUNC(xf86CreateCursorInfoRec)
@@ -1216,5 +1217,8 @@
     /* shadowfb */
     SYMFUNC(ShadowFBInit)
 
+    /* print_edid */
+    SYMFUNC(xf86PrintEDID)
+
     {0, 0}
 };
